src/styles/main.scss

```diff
@@ -1,3 +1,215 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+$blue-color: #00acdc;
+$link-color: #060b35;
+
 body {
+  font-family: "Roboto", sans-serif;
+  box-sizing: border-box;
   margin: 0;
 }
+
+/* Header starts */
+
+.header {
+  background: #fff;
```

Comment:
Order your properties in css due to this order. Checklist, point 12

https://codeguide.academy/html-css.html#css-order